### DISCUSSION TITLE
sharif-dormitory/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#35] [Announcements]
# sharif-dormitory/

شرایط خوابگاه متاهلی دانشگاه شریف برای دانشجویان متاهل

https://aprd.ir/sharif-dormitory/

**Comment by editogoli3** [comment]
خیلی جامع و کاربردی بود واقعا ممنونم بابت زمانی که گذاشتید و این بلاگ مهم و کاربردی رو نوشتید
فقط اگر به موضوع امنیت و محله خوابگاه هم اشاره میکردید دیگه واقعا هیچی کم نداشت
بازم ممنون

**Reply by pourmand1376** [reply]
سلام. وقت شما بخیر. 

امنیت خود خوابگاه رو که گفتم. خدا رو شکر خود خوابگاه نگهبان ۲۴ ساعته داره. دور تا دورش هم دوربین داره و خلاصه امن هست. 

امنیت اطراف خوابگاه و محله هم در کل خوبه. البته فکر می‌کنم، هیچ‌ قسمتی از تهران رو نمیشه کاملاً امن در نظر گرفت. 

من یادمه که بعضی صبح‌ها تقریباً ساعت ۶ می‌رفتم نانوایی محله نون می‌گرفتم و شب‌ها هم اکثراً ساعت ۱۲ شب پیاده‌روی می‌رفتم‌ و احساس ناامنی هم نداشتم.

البته چند تا سگ اون اطراف بودند که سر و صدا می‌کردند ولی بی‌زبون‌ها اصلاً کاری به ما نداشتند. 

با توجه به شناختی که از محله و ساکنانش پیدا کردم، فکر نمی‌کنم از سمت ساکنان مشکلی برای شما پیش بیاد. ولی بالاخره مثل بقیه جاهای تهران، ممکنه یکی از یه جا دیگه بیاد و مثلاً خفت‌گیری بکنه.

**Reply by editogoli3** [reply]
آقا من واقعا نمیدونم چجوری تشکر کنم از ذوق و زمانی که در نوشتن این مقاله کامل به کار بردید
بازم ممنون

**Comment by erfanesteghamat** [comment]
سلام وقت بخیر
ممنونم بابت نوشتن تجربیاتتون.
در رابطه با خود واحد، شما قفل واحد رو عوض کردین؟ تاسیسات عوض می‌کنه؟
برای اجاق گاز فضای کافی هست؟ خودتون چه نوع اجاق گازی استفاده کردید؟

**Reply by pourmand1376** [reply]
سلام. وقت شما هم بخیر و خوشی. 

واحد کامل تحویل شما داده میشه. اگر دوست داشتید می‌تونید قفلش رو عوض کنید و می‌تونید با تأسیسات هم هماهنگ کنید که براتون انجامش بدن. ولی کار ساده‌ای هست، خودتون هم می‌تونید انجام بدید. به شخصه اصلاً قفل رو عوض نکردم. 

ولی بعداً که از اونجا رفتم به خونه جدید، متوجه شدم که رسم تهران اینه که همه یک عدد قفل همراه خودشون دارند و هر جا میرن قفل خودشون رو می‌برند. اینه که اصلاً چیز عجیبی نیست.

برای اجاق گاز هم فضای کافی هست، ما خودمون گاز فر داشتیم. اما برای لباسشویی و ظرفشویی داخل واحد جا نداره. لباس‌شویی رو میشه داخل اتاق کولر قرار داد. اونجا جا به اندازه کافی هست. خصوصاً این که اتاق کولر رو تعمیر کردند و کولر رو بردند بالای اتاق قرار دادند. البته همچنان رطوبت کولر ممکنه لباسشویی رو خراب کنه. 

بعضی از بچه‌ها هم با هزینه شخصی خودشون تعمیرکار آورده بودند و برای لباسشویی و ظرف‌شویی داخل آشپزخونه جا باز کرده بودند. ولی حالت عادی‌اش جا برای هیچ‌کدوم از این دو تا نیست.
